Branch2 Remove BESBasicInterface

jjeichert opened the pull request.

jgallagher59701 commented:
Jim, you want to merge master into a branch you're working on? That seems fine to me. Generally, you only need to ask about the other direction (merging a branch you're working on into master), but it's fine to ask.

James

jjeichert commented:
I created branch2 from the master and then made my changes.  I then did the following:

> git add <4 files>
> git commit
> git push –u origin branch2

On branch branch2
Your branch is ahead of 'origin/master' by 1 commit.
  (use "git push" to publish your local commits)
Changes not staged for commit:
                modified:   modules/hdf4_handler (new commits)
                modified:   modules/hdf5_handler (new commits)

Untracked files:
                dispatch/make2.out
                make.out
                make1.out
                make2.out

no changes added to commit

> git push -u origin branch2

To https://github.com/opendap/bes.git
! [rejected]        branch2 -> branch2 (fetch first)
error: failed to push some refs to 'https://github.com/opendap/bes.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

Jim

From: James Gallagher <notifications@github.com>
Reply-To: OPENDAP/bes <reply@reply.github.com>
Date: Friday, January 27, 2017 at 6:10 PM
To: OPENDAP/bes <bes@noreply.github.com>
Cc: "Eichert, James (GSFC-423.0)[RAYTHEON CO]" <james.eichert@nasa.gov>, Author <author@noreply.github.com>
Subject: Re: [OPENDAP/bes] Branch2 Remove BESBasicInterface (#32)


Jim, you want to merge master into a branch you're working on? That seems fine to me. Generally, you only need to ask about the other direction (merging a branch you're working on into master), but it's fine to ask.

James

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/OPENDAP/bes/pull/32#issuecomment-275799626>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AXVRp8OTsbzQUNQ6vOHUdiBP7NZ4Qfw0ks5rWnlTgaJpZM4LwOLC>.
